float64 degree
int32 time_ms
---
bool success
